### ISSUE TITLE
Typo in id for CursorWordAccessibilityLeftSelect

### ISSUE THREAD

**Issue by gregvanl** [microsoft/vscode#79961]
- VSCode Version: 1.38 Insiders
- OS Version: macOS

export class CursorWordAccessibilityLeftSelect extends WordLeftCommand {
	constructor() {
		super({
			inSelectionMode: true,
			wordNavigationType: WordNavigationType.WordAccessibility,
			id: 'cursorWordAccessibilitLeftSelecty',

id should be 'cursorWordAccessibilityLeftSelect'.
The 'y' is misplaced.
